source/annex/macro.rst

```diff
@@ -209,7 +47,7 @@ The capital value share parameter :math:`\alpha_r` can be interpreted as the opt
 with lower values between 0.24 and 0.28 assumed for developed regions and slightly higher values of 0.3 assumed for economies in transition and developing country regions.
 
 Calibration
```

Comment:
I think this can be moved out of here and into the generic docs. We may want to refer to the original GDP trajectories (Dellink probably?) and just say that we supplied these trajectories to calibration in <refer to new calibration section in ix docs>.